libtenzir/builtins/operators/hash.cpp

```diff
@@ -193,8 +195,37 @@ class plugin final : public virtual operator_plugin<hash_operator> {
   }
 };
 
+class plugin2 : public virtual function_plugin {
+  auto name() const -> std::string override {
+    return "xxhash3";
+  }
+
+  auto make_function(invocation inv,
+                     session ctx) const -> failure_or<function_ptr> override {
+    auto expr = ast::expression{};
+    auto salt = std::optional<std::string>{};
+    TRY(argument_parser2::function("xxhash3")
+          .add(expr, "<field>")
```

Comment:
```suggestion
          .add(expr, "<expr>")
```
When you take an `ast::expression`, you usually don't want to name it `field`, as the point of expressions is to accept more than fields.